packages/xo-server/src/xo-mixins/backups-ng/index.mjs

```diff
@@ -484,6 +485,23 @@ export default class BackupNg {
         })
       }
       return result.id
+    } catch (error) {
+      if (
+        // xxhash is the new form consistency hashing in CH 8.1 which uses a faster,
+        // more efficient hashing algorithm to generate the consistency checks
+        // in order to support larger files without the consistency checking process taking an incredibly long time
+        error.code === 'IMPORT_ERROR' &&
+        error.params?.some(
+          param =>
+            param.includes('INTERNAL_ERROR') &&
+            param.includes('Expected to find an inline checksum') &&
+            param.includes('.xxhash')
+        )
+      ) {
+        log.warn('importVmBackupNg', { error })
+        throw new JsonRpcError('Importing the VM backup requires that CH >= 8.1 be restored.')
+      }
```

Comment:
After discussing with @olivierlambert, I believe it makes sense to move this logic to `xen-orchestra/xapi/vm.js#import` to  handle all XVA imports.

@Rajaa-BARHTAOUI Do you think it can be done easily?

Reply:
Yes, it may makes sense, but it will be complicated to do it for this release.